### ISSUE TITLE
[bug] WebviewWindow X and Y are modified, whilst they shouldn't!

### ISSUE THREAD

**Issue by Miniontoby** [tauri-apps/tauri#6784]
### Describe the bug

Here is the 3 screen layout:
```
[1] [2]
[3]
```
And here the result:
![Screenshot 3 screens](https://media.discordapp.net/attachments/1099699996493889556/1099699996930084984/image.png)

Here is the 4 screen layout:
```
[1] [2] [3] [4]
```
And here the result:
![Screenshot 4 screens](https://cdn.discordapp.com/attachments/1099699996493889556/1099716314169360394/image.png)

See the code in the reproduction:

The output of the `console.log(monitor)`:
```
{ "name": "\\\\.\\DISPLAY1", "scaleFactor": 1.5, "position": { "type": "Physical", "x": 0, "y": 0 }, "size": { "type": "Physical", "width": 1920, "height": 1080 } }
{ "name": "\\\\.\\DISPLAY216", "scaleFactor": 1, "position": { "type": "Physical", "x": 1920, "y": 0 }, "size": { "type": "Physical", "width": 1920, "height": 1080 } }
{ "name": "\\\\.\\DISPLAY217", "scaleFactor": 1, "position": { "type": "Physical", "x": 3840, "y": 0 }, "size": { "type": "Physical", "width": 1920, "height": 1080 } }
{ "name": "\\\\.\\DISPLAY218", "scaleFactor": 1, "position": { "type": "Physical", "x": 5760, "y": 0 }, "size": { "type": "Physical", "width": 1920, "height": 1080 } }
```

The output of the ` console.log(await webviewwindow.innerPosition())`:
```
{ "type": "Physical", "x": 0, "y": 0 }
{ "type": "Physical", "x": 2880, "y": 0 }
{ "type": "Physical", "x": 5760, "y": 0 }
{ "type": "Physical", "x": 8640, "y": 0 }
```
And yeah if we divided the x by the width:
```
2880/1920=1.5
5760/1920=3
8640/1920=4.5
```

So with this info, I think it uses the scale factor (for screen 1 it is 1.5, the rest is 1) that I use because my screen is too small for 100% zoom.

But it doesn't need to use the scaleFactor at all here!

when adding (or in the code below: uncommenting) `webviewwindow.setPosition(monitor.position)` to the `created` event it does correct it, but I don't want to have to do that since that only fixes MY X and Y (not even the width and height) and just only for me and not for ANYONE else!

I was also sort of able to fix it by dividing the x and y of the `monitor.position` by the `appWindow.scaleFactor()` because the program would then multiply it by `appWindow.scaleFactor()`

BUT when the other screen has a different scaleFactor, it didn't work with the width and height... In this case the width and the height were too large values.

I hope we can solve it in the backend code instead of having to solve it in my frontend!

### Reproduction

```ts
async open(monitor: Monitor, version: string, url: string="/") {
  if (id == null) id = monitor.name;
  console.log(monitor)
  let webviewwindow = new WebviewWindow(`presentation${id}`, {url: url, fullscreen: true, resizable: false, title: 'presentation'+version, x: monitor.position.x, y: monitor.position.y});
  webviewwindow.once('tauri://created', async function () {
    console.log(await webviewwindow.innerPosition())
    // When uncommenting the following line, the position is corrected correctly
    // await webviewwindow.setPosition(monitor.position)
  });
}
const monitors: Monitor[] = await availableMonitors();
open(monitors[0], '1')
open(monitors[1], '2')
open(monitors[2], '3')
```

And then look or make a screenshot if you don't use REAL monitors but virtual like I did in combination with a real ones!

### Expected behavior

I expected the windows to be in the correct location with the correct width and height.

### Platform and versions

```text
Environment
  › OS: Windows 10.0.19044 X64
  › Webview2: 112.0.1722.58
  › MSVC:
      - Visual Studio Community 2022
      - Visual Studio Build Tools 2022
  › Node.js: 18.4.0
  › npm: 8.13.2
  › pnpm: 7.25.0
  › yarn: Not installed!
  › rustup: 1.25.1
  › rustc: 1.64.0
  › cargo: 1.64.0
  › Rust toolchain: stable-x86_64-pc-windows-msvc

Packages
  › @tauri-apps/cli [NPM]: 1.2.3
  › @tauri-apps/api [NPM]: 1.2.0
  › tauri [RUST]: 1.2.4,
  › tauri-build [RUST]: 1.2.1,
  › tao [RUST]: 0.15.8,
  › wry [RUST]: 0.23.4,

App
  › build-type: bundle
  › CSP: unset
  › distDir: ../build
  › devPath: http://localhost:5173/
  › framework: Svelte
  › bundler: Vite

App directory structure
  ├─ .svelte-kit
  ├─ .vscode
  ├─ build
  ├─ node_modules
  ├─ songs
  ├─ src
  ├─ src-tauri
  └─ static
```

**Comment by amrbashir** [comment]
This should be solved once https://github.com/tauri-apps/tauri/pull/5381 lands so I will close this as duplicate of https://github.com/tauri-apps/tauri/issues/5228

**Comment by Miniontoby** [comment]
But #5381 is commented about a year ago! I would like this bug fixed asap

Is it possible to use the PR's code in my project (and yeah it is possibly outdated a lot... but still)? And then mostly: How could I do that?
